src/commands/new-proj/new-repo.ts

```diff
@@ -24,12 +19,10 @@ async function action(args) {
   let publicity : string = args.options.publicity ?
     args.options.publicity :
     "private";
-
-  let givenPath : string =  args.options.path ?
-    args.options.path :
-    defaultPath;
-
-  if (!publicity || !repoName) {
+  
+  let givenPath : string =  args.options.path;
```

Comment:
program should use default path if none is given

Reply:
now conditionally uses default path